<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P_PictureCombiner</class>
 <widget class="QWidget" name="P_PictureCombiner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1" columnstretch="2,3">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>预览图</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>图片总宽高：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_totalPic_wh">
           <property name="text">
            <string>100x100</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_sizeTip_2">
           <property name="styleSheet">
            <string notr="true">color: rgb(154, 154, 154);</string>
           </property>
           <property name="text">
            <string>◆ rmmv的动画资源标准为每帧192x192大小，通常为5列。</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_preview" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="styleSheet">
         <string notr="true">
QToolButton{
	padding:2px;
	border: 1px solid transparent;
	background: rgba(255,255,255,0);
}
QToolButton:hover{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #fdecad, stop:0.3 #fde48c, stop:0.9 #fce58c, stop:1 #fdfadf )
}
QToolButton:pressed{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: #FDF4BF;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_zoom_btn" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_gridLine">
           <property name="text">
            <string>辅助网格线</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>矩阵</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,1">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>列数：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_column">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_column">
        <property name="text">
         <string>自适应</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_row">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBox_row">
        <property name="text">
         <string>自适应</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>行数：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>顺序：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>从左往右</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>从上往下</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>帧属性</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,2,1,0">
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_frameWidth">
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="label_sizeTip">
        <property name="styleSheet">
         <string notr="true">color: rgb(154, 154, 154);</string>
        </property>
        <property name="text">
         <string>◆ 由于帧之间的资源宽高不一致，所以默认值取最大值。</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBox_frameHeightDefault">
        <property name="text">
         <string>默认值</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_frameHeight">
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>帧固定宽度：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_frameWidthDefault">
        <property name="text">
         <string>默认值</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>帧固定高度：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QWidget" name="widget_2" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../drillgifactionsequenceeditor.qrc"/>
 </resources>
 <connections/>
</ui>
